calicam:
  ros__parameters:
    calibration_file: ""
    fov: 100.0
    fps: 30.0
    camera_index: -1
    undistort_rectify: true
    monochrome: false